muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/announcer/body/delete-announcement-dialog.tsx

```diff
@@ -66,13 +66,13 @@ class DeleteAnnouncementDialog extends React.Component<DeleteAnnouncementDialogP
     let footer = (closeDialog: ()=>any)=>{
       return (          
          <div className="dialog__button-set">
-          <Link className="button button--cancel button--standard-cancel" onClick={closeDialog}>
-            {this.props.i18n.text.get('plugin.announcer.deleteDialog.cancelButton.label')}
-          </Link>
-          <Link className="button button--fatal button--standard-ok"
-            onClick={this.deleteAnnouncement.bind(this, closeDialog)} disabled={this.state.locked}>
+          <Link className="button button-dialog--execute"
```

Comment:
This seems to be wrong regarding BEM. Button and button-dialog are both BEM blocks and it seems they are used as mixed blocks but button-dialog block has only modifier classname which is wrong.

Either use button or button-dialog only and then add modifier with base block name ie. "button-dialog button-dialog--execute".